docs/time/index.rst

```diff
@@ -997,39 +1082,43 @@ calculations following different IAU resolutions.  Sample usage::
   >>> t.sidereal_time('apparent', '-90d', 'IAU1994')  # doctest: +FLOAT_CMP +REMOTE_DATA
   <Longitude 23.08950365 hourangle>
 
+.. EXAMPLE END
+
 Time Deltas
 -----------
 
-Simple time arithmetic is supported using the |TimeDelta| class.  The
-following operations are available:
+Time arithmetic is supported using the |TimeDelta| class. The following
+operations are available:
 
-- Create a TimeDelta explicitly by instantiating a class object
-- Create a TimeDelta by subtracting two Times
-- Add a TimeDelta to a Time object to get a new Time
-- Subtract a TimeDelta from a Time object to get a new Time
-- Add two TimeDelta objects to get a new TimeDelta
-- Negate a TimeDelta or take its absolute value
-- Multiply or divide a TimeDelta by a constant or array
-- Convert TimeDelta objects to and from time-like Quantities
+- Create a |TimeDelta| explicitly by instantiating a class object.
+- Create a |TimeDelta| by subtracting two |Time| objects.
+- Add a |TimeDelta| to a |Time| object to get a new |Time|.
+- Subtract a |TimeDelta| from a |Time| object to get a new |Time|.
+- Add two |TimeDelta| objects to get a new |TimeDelta|.
+- Negate a |TimeDelta| or take its absolute value.
+- Multiply or divide a |TimeDelta| by a constant or array.
+- Convert |TimeDelta| objects to and from time-like |Quantities|.
 
 The |TimeDelta| class is derived from the |Time| class and shares many of its
-properties.  One difference is that the time scale has to be one for which one
-day is exactly 86400 seconds.  Hence, the scale cannot be UTC.
+properties. One difference is that the time scale has to be one for which one
+day is exactly 86400 seconds. Hence, the scale cannot be UTC.
 
 The available time formats are:
 
 =========  ===================================================
-Format            Class
+Format     Class
 =========  ===================================================
 sec        :class:`~astropy.time.TimeDeltaSec`
 jd         :class:`~astropy.time.TimeDeltaJD`
 datetime   :class:`~astropy.time.TimeDeltaDatetime`
 =========  ===================================================
 
 Examples
-^^^^^^^^^
+""""""""
```

Comment:
I think this should stay as "^"'s, right?

Reply:
Oops yes